packages/edit-post/src/components/visual-editor/style.scss

```diff
@@ -63,9 +63,10 @@
 	margin-left: auto;
 	margin-right: auto;
 
-	// Space title similarly to other blocks.
-	// This uses negative margins so as to not affect the default block margins.
-	margin-bottom: -$block-padding - $block-spacing - $border-width - $border-width;
+	// Apply default block margin below the post title.
+	// This ensures the first block on the page is almost in a good position.
```

Comment:
> This ensures the first block on the page is almost in a good position.

Super-minor: Is "almost" the word we want to use here? It seems like that just opens up more questions. 😄